### PR TITLE
[Connectedk8s] Updated to GA sdk

### PR DESCRIPTION
**Pull request by alphaWizard**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Connectedk8s

**Comment by alphaWizard** [comment]
Pending testing in EUAP.

**Review by fengzhou-msft — APPROVED** [review]

File: src/connectedk8s/azext_connectedk8s/vendored_sdks/models/connected_cluster_patch_py3.py [hunk 1]
```diff
@@ -17,18 +17,17 @@ class ConnectedClusterPatch(Model):
...
-     Azure.
-    :type agent_public_key_certificate: str
+    :param properties: Describes the connected cluster resource properties
+     that can be updated during PATCH operation.
+    :type properties: object
     """
 
     _attribute_map = {
         'tags': {'key': 'tags', 'type': '{str}'},
-        'agent_public_key_certificate': {'key': 'properties.agentPublicKeyCertificate', 'type': 'str'},
```
Comment (akashkeshari) [inline src/connectedk8s/azext_connectedk8s/vendored_sdks/models/connected_cluster_patch_py3.py:28]: why is this removed?

**Review by akashkeshari — APPROVED** [review]

File: src/connectedk8s/azext_connectedk8s/vendored_sdks/models/connected_cluster_patch_py3.py (hunk 1 above)
Comment (alphaWizard) [reply src/connectedk8s/azext_connectedk8s/vendored_sdks/models/connected_cluster_patch_py3.py:28]: This is as per auto-generated from GA swagger. Discussed offline.
